packages/taco/test/conditions/context.test.ts

```diff
@@ -122,10 +123,11 @@ describe('context', () => {
     it('detects if a signer is required', () => {
       const conditionObj = {
         ...testContractConditionObj,
+        parameters: [USER_ADDRESS_PARAM],
         returnValueTest: {
-          ...testReturnValueTest,
-          value: USER_ADDRESS_PARAM,
-        },
+          comparator: '==',
+          value: 100,
+        } as ReturnValueTestProps,
```

Comment:
Should this test could be broken into 2?
1) detects if signer is required because context var only in parameters
2) detects if signer is required because content vari only in returnValueTest